styles/aboutus.css

```diff
@@ -0,0 +1,164 @@
+:root {
+    --background-color: #FAF4E8;
+    --feature-color: #34959E;
+    --footer-color: #9D6034;
+    --nav-bar: #DAA46D;
+    --text-color: #382B22;
+}
+
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    background-color: var(--background-color);
+    font-family:Arial, Helvetica, sans-serif;
+    height: 90%;
+    margin: 0;
+    padding: 0;
+    width: 100%;
+}
+
+h2 {
+    margin: 0;
+}
+
+h3 {
+    margin: 0;
+}
+p {
+    margin: 0;
+}
+
+.header {
+    
+}
+
+.nav-bar {
+    /* margin: 32px 0 0 32px; */
+    background-color: #313640;
+    background-color: var(--nav-bar);
+    border-radius: 25px 25px 0 0;
+    display: flex;
+    flex-direction: column;
+    overflow-x: hidden;
+    overflow-y: auto; 
+    padding-top: 32px;
+    width: 240px;
+}
+
+.main {
+    /* width: 100%; */
+    background-color: var(--color-background);
+    display: flex;
+    gap: 32px;
+    height: calc(100% - 350px);
+    margin: 150px 32px 32px 32px;
+}
+
+
+.team-tabs {
+    /* gap: 18px; */
+    display: flex;
+    flex-direction: column;
+    gap: 32px;
+}
+
+    .tab {
+           
+    }
+
+    .tab,
+    .tab.active {
+        margin: 0;
+        font-size: 1rem;
+        text-align: center;
+        color: white;
+        font-size: 1rem;
+        gap: 32px;
+        text-decoration: none;
+        display: block;
+}   
+
+    .tab:hover {
+        color: var(--feature-color);
+        cursor: pointer;
+        transition: color 0.3s ease, color 0.3s ease;
+    }
+
+    .main-container {
+        display: flex;
+        min-height: calc(100dvh - (383px + 96px));
+        margin-top: 6rem;
+      }
+
+.team {
+    background-color: var(--background-color);
+    display: grid;
+    grid-template-columns: 1fr 2fr;
+    grid-template-rows: auto auto auto 1fr;
+    margin: 1rem;
+}
+
+.img-info__content {
+    grid-column: 1;
+}
+    .image {
+        grid-row: 1 / span 2;
+    }
+        .image__child {
+            width: 100%;
+        }
+
```

Comment:
Think about formatting the file so the code-structure becomes more readable, other than that you have quite some nice code! 😄